perfil/person/management/commands/load_pinfo.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management.base import BaseCommand
+
+from tqdm import tqdm
+
+from perfil.person.models import Person, PersonInformation
+
+
+class Command(BaseCommand):
+
+    help = ""
+    model = PersonInformation
+    bulk_size = 2 ** 10
+    slice_csv = False
+
+    def handle(self, *args, **options):
+        desc = 'Creating Information table'
+        total = Person.objects.count()
+        people = Person.objects.all()
+
+        position_congress = ['15', '16', '17', '18', '25', '26', '27', '28']
+
+        with tqdm(total=total, desc=desc, unit='people') as progress:
+            infos = []
+            count = 0
+            for p in people:
+                election_parties = p.election_parties
+                filiation_parties = p.filiation_parties
+                parties_change = len(set(filiation_parties + election_parties))
+
+                elections_results = list(p.elections.values_list('result',
+                                                                 flat=True))
+                num_elections = len(elections_results)
+                elected_by_quota = elections_results.count('2')
+                elected = elections_results.count('1')
+                total_elections_won = elected_by_quota + elected
+
+                congressperson = False
+                elections = list(p.elections.values_list('position', 'result'))
+                for election in elections:
+                    position = election[0]
+                    result = election[1]
+                    if position in position_congress and result in ['1', '2']:
```

Comment:
What about avoiding magical indexes?

```python
for position, result in elections:
    if position in position_congress and result in ['1', '2']:
```

And what does these numbers `1` and `2` mean?